manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: MoheshwarAmarnathBiswas.UsefulToolsForWindows
+PackageVersion: 1.1

```

```suggestion
PackageVersion: 1.2
```
Version in control panel is 1.2